rest/src/main/java/org/n52/web/ctrl/ParameterController.java

```diff
@@ -132,22 +135,48 @@ public void getRawData(HttpServletResponse response,
     }
 
     @Override
-    public ModelAndView getCollection(String locale, MultiValueMap<String, String> query) {
+    public ModelAndView getCollection(HttpServletRequest request, HttpServletResponse response, String locale, MultiValueMap<String, String> query) {
         RequestUtils.overrideQueryLocaleWhenSet(locale, query);
         IoParameters queryMap = QueryParameters.createFromQuery(query);
         LOGGER.debug("getCollection() with query '{}'", queryMap);
-
+        OutputCollection<T> result;
+        
         if (queryMap.isExpanded()) {
             Stopwatch stopwatch = Stopwatch.startStopwatch();
-            OutputCollection<T> result = addExtensionInfos(parameterService.getExpandedParameters(queryMap));
+            result = addExtensionInfos(parameterService.getExpandedParameters(queryMap));
             LOGGER.debug("Processing request took {} seconds.", stopwatch.stopInSeconds());
-            return createModelAndView(result);
         } else {
-            OutputCollection<T> results = parameterService.getCondensedParameters(queryMap);
-            return createModelAndView(results);
+            result = parameterService.getCondensedParameters(queryMap);
+        }
+        
+        // Add Paging if requested
+        if (queryMap.containsParameter("limit") || queryMap.containsParameter("offset")){
```

Comment:
could you please adding paging to [ParameterRequestMappingAdapter](https://github.com/52North/series-rest-api/blob/develop/rest/src/main/java/org/n52/web/ctrl/ParameterRequestMappingAdapter.java#L52) .. I think it is better placed here as it is enhancing and auto handling requests